### PR TITLE
release v2020-01-20

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
cc @matentzn

### CODE REVIEW

**Comment by matentzn** [comment]
There is a huge diff on cl.owl and cl.obo, but it looks to me like this is because for the first time ever, they are correct; please make sure @dosumis to review these artefacts thoroughly this time (I checked everything else, and it looks fine). Also look at cl-base.obo just to track real changes.

I think the enormous diff on cl.obo and cl.owl (you can only view them locally, by checking out the branch and then looking at the history in github desktop, or using CLI) is basically the difference between cl-full (it looks to me like it was full before!) and cl-basic (that's what it looks to me now). I cant understand why it works only now; I can only imagine this is because some files cached on my machine caused mayhem. 

So basically, check that, make sure you want cl-basic as the main release, and look at the diff for sure. I don't know why that happened, but no matter why, it seems to be correct now!

**Comment by matentzn** [comment]
Do not merge: https://github.com/obophenotype/cell-ontology/pull/802 merge that first

**Review by matentzn — APPROVED** [review]
